src/main/java/com/puppycrawl/tools/checkstyle/checks/imports/CustomImportOrderCheck.java

```diff
@@ -525,10 +525,8 @@ public void beginTree(DetailAST rootAST) {
     @Override
     public void visitToken(DetailAST ast) {
         if (ast.getType() == TokenTypes.PACKAGE_DEF) {
-            if (customImportOrderRules.contains(SAME_PACKAGE_RULE_GROUP)) {
                 samePackageDomainsRegExp = createSamePackageRegexp(
```

Comment:
If the if statement is being removed, the indentation must be updated too. Please decrease the lines inside the old if statement by 4 spaces.

Reply:
done